CRUDRecipeEF.BL.DL/Entities/Restaurant.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CRUDRecipeEF.BL.DL.Entities
+{
+    public class Restaurant
+    {
+        public int Id { get; set; }
+        public string Name { get; set; }
```

Comment:
Required

Reply:
I did it and when I added migration it poped this message "An operation was scaffolded that may result in the loss of data. Please review the migration for accuracy"
Do I continue?

Reply:
Yeah continue

Reply:
I added the required. I think its ready